File: rails-demo/app/controllers/pages_controller.rb
class PagesController < ApplicationController
  def account_settings
    @title = 'Account settings - Pages'
  end

  def chat
    @title = 'Chat - Pages'
    render layout: "main/layout-2-flex"
  end

  def clients
    @title = 'Clients - Pages'
    render layout: "main/layout-2-flex"
  end

  def contacts
    @title = 'Contacts - Pages'
  end

  def faq
    @title = 'FAQ - Pages'
  end

  def file_manager
    @title = 'File manager - Pages'
  end

  def gallery
    @title = 'Gallery - Pages'
  end

  def help_center
    @title = 'Help center - Pages'
  end

  def invoice_print
    @title = 'Invoice (Print version) - Pages'
    render layout: "main/layout-blank"
  end

  def invoice
    @title = 'Invoice - Pages'
  end

  def kanban_board
    @title = 'Kanban board - Pages'
  end

  def pricing
    @title = 'Pricing - Pages'
  end

  def profile_v1
    @title = 'Profile v1 - Pages'
  end

  def profile_v2
    @title = 'Profile v2 - Pages'
  end

  def search_results
    @title = 'Search results - Pages'
  end

  def task_list
    @title = 'Task list - Pages'
  end

  def teams
    @title = 'Teams - Pages'
  end

  def vacancies
    @title = 'Vacancies - Pages'
  end

  def voting
    @title = 'Voting - Pages'
  end

end

File: laravel-demo/app/Http/Controllers/PagesController.php
<?php

namespace App\Http\Controllers;

use App\User;
use App\Http\Controllers\Controller;

class PagesController extends Controller
{
    public function accountSettings()
    {
        return view('pages.account-settings', ['title' => 'Account settings - Pages']);
    }

    public function chat()
    {
        return view('pages.chat', ['title' => 'Chat - Pages']);
    }

    public function clients()
    {
        return view('pages.clients', ['title' => 'Clients - Pages']);
    }

    public function contacts()
    {
        return view('pages.contacts', ['title' => 'Contacts - Pages']);
    }

    public function faq()
    {
        return view('pages.faq', ['title' => 'FAQ - Pages']);
    }

    public function fileManager()
    {
        return view('pages.file-manager', ['title' => 'File manager - Pages']);
    }

    public function gallery()
    {
        return view('pages.gallery', ['title' => 'Gallery - Pages']);
    }

    public function helpCenter()
    {
        return view('pages.help-center', ['title' => 'Help center - Pages']);
    }

    public function invoicePrint()
    {
        return view('pages.invoice-print', ['title' => 'Invoice (Print version) - Pages']);
    }

    public function invoice()
    {
        return view('pages.invoice', ['title' => 'Invoice - Pages']);
    }

    public function kanbanBoard()
    {
        return view('pages.kanban-board', ['title' => 'Kanban board - Pages']);
    }

    public function pricing()
    {
        return view('pages.pricing', ['title' => 'Pricing - Pages']);
    }

    public function profileV1()
    {
        return view('pages.profile-v1', ['title' => 'Profile v1 - Pages']);
    }

    public function profileV2()
    {
        return view('pages.profile-v2', ['title' => 'Profile v2 - Pages']);
    }

    public function searchResults()
    {
        return view('pages.search-results', ['title' => 'Search results - Pages']);
    }

    public function taskList()
    {
        return view('pages.task-list', ['title' => 'Task list - Pages']);
    }

    public function teams()
    {
        return view('pages.teams', ['title' => 'Teams - Pages']);
    }

    public function vacancies()
    {
        return view('pages.vacancies', ['title' => 'Vacancies - Pages']);
    }

    public function voting()
    {
        return view('pages.voting', ['title' => 'Voting - Pages']);
    }

}
